packages/repco-webcomponent/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "repco-recommander-box",
```

Comment:
should be `repco-webcomponent` or rename the package folder, but i think `repco-webcomponent` is good